plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -1976,14 +1976,11 @@ else if (object.getLivingEntity() instanceof Steerable) {
         // @attribute <EntityTag.is_sleeping>
         // @returns ElementTag(Boolean)
         // @description
-        // Returns whether the player, NPC, or villager is currently sleeping.
+        // Returns whether a living entity is currently sleeping.

```

Comment:
What does this return for entities that can't sleep?

Also if the mech name matches the tag name you need an `@mechanism` in the meta